coreos-base/oem-gce/files/files/google-cloud-sdk.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+alias gcloud="(docker images google/cloud-sdk || docker pull google/cloud-sdk) > /dev/null;docker run -ti --rm --net="host" -v $HOME/.config:/root/.config -v /var/run/docker.sock:/var/run/docker.sock google/cloud-sdk gcloud"
+alias gsutil="(docker images google/cloud-sdk || docker pull google/cloud-sdk) > /dev/null;docker run -ti --rm --net="host" -v $HOME/.config:/root/.config google/cloud-sdk gsutil"
```

Comment:
Maybe drop the quotes from `--net="host"`.  They aren't needed, and they unquote the alias string.  It makes no difference in this case, but it's reminiscent of actual problems.

Reply:
Done.